### PR TITLE
Fix various empty states

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

Just some here and there that has bothered me

# Result

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

<table>
    <tr>
        <td>Before</td>
        <td>After</td>
    </tr>
    <tr>
        <td>
            image
        </td>
        <td>
            image
        </td>
    </tr>
    <tr>
        <td>
            image
        </td>
        <td>
            image
        </td>
    </tr>
    <tr>
        <td>
            image
        </td>
        <td>
            image
        </td>
    </tr>
    <tr>
        <td>
            image
        </td>
        <td>
            image
        </td>
    </tr>
</table>

# Testing

- [x] I have thoroughly tested my changes.

See images above.

### CODE REVIEW

File: app/routes/forum/components/ForumList.tsx [hunk 1]
```diff
@@ -30,11 +33,21 @@ const ForumList = () => {
...
+      {!forums.length && (
+        <EmptyState
+          iconNode={<MessageSquareDashed />}
+          header="Her var det tomt ..."
+          body="Opprett et nytt forum da vel!"
+        />
+      )}
+      {forums.length > 0 && (
+        <Flex column>
+          {forums?.map((f: PublicForum) => (
```
Comment (norbye) [inline app/routes/forum/components/ForumList.tsx:46]:
```suggestion
          {forums?.map((forum: PublicForum) => (
```
Not your doing, but spelling it out would be kinda nice(:

**Review by norbye — APPROVED** [review]
Sweet!

File: app/routes/forum/components/ForumList.tsx (hunk 1 above)
Comment (ivarnakken) [reply app/routes/forum/components/ForumList.tsx]: Fair enough! Fixed 🦖

**Review by eikhr — APPROVED** [review]
Very nice!

**Pull request merged**